src/geocat/viz/contourf/_add_geo_features.py

```diff
@@ -0,0 +1,82 @@
+import cartopy.feature as cfeature
+
+
+class add_geo_features:
+    """Class to add geographical features to a figure.
+
+    Kwargs:
+
+        show_coastline (:obj:`bool`): Whether to show coastlines in figure. Default False, unless a projection is specified.
+
+        show_lakes (:obj:`bool`): Whether to show lakes in figure. Default False.
+
+        show_land (:obj:`bool`): Whether to show land in figure. Default false.
+    """
+
+    def _init_(self, *args, **kwargs):
```

Comment:
@anissa111 @erinlincoln just a quick question for my own knowledge, does reusing ```_init_```  or ```__init___``` cause problems with the code? PyCharm is referencing this line and 53 in _set_up_fig as "overrides method in (file)". In _plot_util and contourf, same thing is noted except for ```__init__```. Is this an issue that we need to look into for naming purposes, or can we just overlook it?

Reply:
So sort of? __inits__ are separate for the different classes, but I believe they are overriding each other here because they're titled `_init_` instead of `__init__`.

They're set up as parent and child class here (`_geo_axes being parent,` `_fix_axes` being child), and to do that in python, both need to have `__init__` functions, and the parent's `__init__` can be called from the child's `__init__`. Since they aren't being picked up as the "proper" init functions, though, the child class *is* overriding the improper parent init, though.

Reply:
Ok that makes sense. I just experimented with adding the extra _ to this file and _set_up_fig and while it still says there is an override, it wants the overridden file to be selected from a drop down menu.

Reply:
I think since I combined all the parent classes into NCL_Plot, this shouldn't be a problem unless the contourf init is overriding the NCL_Plot init or vice versa?